### PR TITLE
modify module

### PR DESCRIPTION
**Pull request by matiasmarrapodi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,333 +1,83 @@
...
-## Makefile Targets
-```text
-Available targets:
-
-  help                                Help screen
-  help/all                            Display help for all targets
-  help/short                          This help short screen
-  lint                                Lint terraform code
+```hcl
+module "permission_sets" {
```
Comment (ivours) [inline README.md]: Shouldn't this module instance be named something like "sso"?

File: README.md [hunk 2]
```diff
@@ -1,333 +1,83 @@
...
+  name   = "S3Access"
+  path   = "/"
+  policy = data.aws_iam_policy_document.S3Access.json
+  tags   = module.this.tags
+}
+
+account_assignments = [
+    {
+        account = "111111111111",
+        permission_set_arn = module.main.permission_sets["SSO_PS_ADMINISTRATOR_STAGE"].arn,
```
Comment (ivours) [inline README.md]: Where is the instantiation of a module named `main`?

File: README.md [hunk 3]
```diff
@@ -1,333 +1,83 @@
...
+}
+
+resource "aws_iam_policy" "S3Access" {
+  name   = "S3Access"
+  path   = "/"
+  policy = data.aws_iam_policy_document.S3Access.json
+  tags   = module.this.tags
+}
+
+account_assignments = [
```
Comment (ivours) [inline README.md:39]: Shouldn't this assignation be inside the module instantiation? (It's a module input)

**Review by ivours — CHANGES_REQUESTED** [review]
Please review the comments I've left in the code.

Thanks!

File: README.md (hunk 1 above)
Comment (matiasmarrapodi) [reply README.md]: I've changed the name to sso

File: README.md (hunk 2 above)
Comment (matiasmarrapodi) [reply README.md]: I replaced it with sso

**Review by ivours — APPROVED** [review]

**Pull request merged**